SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs

```diff
@@ -18,26 +18,25 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Threading;
 using System;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using SonarQube.Client.Models.ServerSentEvents.ClientContract;
 using SonarQube.Client.Models.ServerSentEvents.ServerContract;
 using System.Collections.Generic;
-using System.Threading.Channels;
 using SonarQube.Client.Logging;
 
 namespace SonarQube.Client.Models.ServerSentEvents
 {
     public interface ISSEStreamReader
     {
         /// <summary>
+        /// Wraps the stream response from the server, reads from it and converts it to <see cref="IServerEvent"/>.
         /// Will block the calling thread until an event exists or the connection is closed.
-        /// Can throw an exception if the event is not a valid <see cref="IServerEvent"/>
         /// </summary>
         /// <returns>
-        /// Will return null if the underlying event is unsupported or if there was a problem parsing it.
+        /// Can return null (i.e. if the underlying event type is unsupported).
+        /// Will throw if there was a problem reading from the underlying stream.
```

Comment:
Is this true?

Reply:
Also noticed it, right now no, but I think it should. It's what we talked about yesterday, I think SLVS will need to handle it and make a new session.
I'll remove the comment for now but I think we should throw it.

Reply:
Comment out the comment :-P